src/main/java/org/jabref/logic/exporter/ExporterFactory.java

```diff
@@ -36,21 +35,21 @@ public static ExporterFactory create(Map<String, TemplateExporter> customFormats
         List<Exporter> exporters = new ArrayList<>();
 
         // Initialize build-in exporters
-        exporters.add(new TemplateExporter("HTML", "html", "html", null, FileType.HTML, layoutPreferences, savePreferences));
-        exporters.add(new TemplateExporter(Localization.lang("Simple HTML"), "simplehtml", "simplehtml", null, FileType.HTML, layoutPreferences, savePreferences));
-        exporters.add(new TemplateExporter("DocBook 4.4", "docbook", "docbook", null, FileType.XML, layoutPreferences, savePreferences));
-        exporters.add(new TemplateExporter("DIN 1505", "din1505", "din1505winword", "din1505", FileType.RTF, layoutPreferences, savePreferences));
-        exporters.add(new TemplateExporter("BibO RDF", "bibordf", "bibordf", null, FileType.RDF, layoutPreferences, savePreferences));
-        exporters.add(new TemplateExporter(Localization.lang("HTML table"), "tablerefs", "tablerefs", "tablerefs", FileType.HTML, layoutPreferences, savePreferences));
-        exporters.add(new TemplateExporter(Localization.lang("HTML list"), "listrefs", "listrefs", "listrefs", FileType.HTML, layoutPreferences, savePreferences));
-        exporters.add(new TemplateExporter(Localization.lang("HTML table (with Abstract & BibTeX)"), "tablerefsabsbib", "tablerefsabsbib", "tablerefsabsbib", FileType.HTML, layoutPreferences, savePreferences));
-        exporters.add(new TemplateExporter("Harvard RTF", "harvard", "harvard", "harvard", FileType.RDF, layoutPreferences, savePreferences));
-        exporters.add(new TemplateExporter("ISO 690 RTF", "iso690rtf", "iso690RTF", "iso690rtf", FileType.RTF, layoutPreferences, savePreferences));
-        exporters.add(new TemplateExporter("ISO 690", "iso690txt", "iso690", "iso690txt", FileType.TXT, layoutPreferences, savePreferences));
-        exporters.add(new TemplateExporter("Endnote", "endnote", "EndNote", "endnote", FileType.TXT, layoutPreferences, savePreferences));
-        exporters.add(new TemplateExporter("OpenOffice/LibreOffice CSV", "oocsv", "openoffice-csv", "openoffice", FileType.CSV, layoutPreferences, savePreferences));
-        exporters.add(new TemplateExporter("RIS", "ris", "ris", "ris", FileType.RIS, layoutPreferences, savePreferences).withEncoding(StandardCharsets.UTF_8));
-        exporters.add(new TemplateExporter("MIS Quarterly", "misq", "misq", "misq", FileType.RTF, layoutPreferences, savePreferences));
+        exporters.add(new TemplateExporter(FileType.HTML.getDescription(), "html", "html", null, FileType.HTML, layoutPreferences, savePreferences));
```

Comment:
It appears that now the description is always the description of the file type, right? Then it makes sense to create a new constructor of `TemplateExporter` that automatically sets the description.

Reply:
I thought so, too but on the other hand we have the constructor needed for the custom exporter, so in the end it's the same

Reply:
but at least we could add a new constructor, so that this list gets more readable, right?